pkg/cloudprovider/aws/aws.go

```diff
@@ -30,6 +30,14 @@ const (
 	tagKey = "k8s.io/atlassian-escalator/enabled"
 	// tagValue is the value for the tag applied to ASGs and Fleet requests
 	tagValue = "true"
+	// maxTerminateInstancesTries is the maximum number of times terminateOrphanedInstances can be called consecutively
+	maxTerminateInstancesTries = 3
+	// The TerminateInstances API only supports terminating 1000 instances at a time
+	terminateBatchSize = 1000
+)
+
+var (
+	terminateInstancesTries = 0
```

Comment:
Unsure about the need of this being a global shared variable, would prefer this being owned by the CloudProvider or NodeGroup struct, and passed in to the terminateOrphanedInstances function as an argument